index.html

```diff
@@ -1,16 +1,108 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css">
 
+    <title>Anu's Cake Co.</title>
+</head>
 <body>
-    <!-- Add your markup here -->
-</body>
+    <header>
+        <div class="main">
+        <div class="logo">
+            <img src="https://m.media-amazon.com/images/I/51AIKwexFkL._AC_US100_.jpg">
+            <h1>The best cake in town delivered to your door.</h1>
+            </div>
+          
+            </div>
+    
+    <ul>   
+    <li><a href="#"> HOME</a></li>
+    <li><a href="#"> CAKES</a></li>
+    <li><a href="#"> ORDERING</a></li>
+    <li><a href="#"> LESSON</a></li>
+    <li><a href="#"> ABOUT</a></li>
+</ul> 
+        </div>
+        
+  
+   </header>
+   
+
+<section class="types">
+    <div class="type">
+        <img src="https://tinyurl.com/mr39vhw5">
+<h3>Lets celebrate together</h3>
+<Button>Order</Button>
+</div>
+
```

Comment:
I think there should be a section to separate the whole content from other sections.